extensions/parquet/table/src/main/java/io/deephaven/parquet/table/util/ByteBufferAsyncResponseTransformer.java

```diff
@@ -0,0 +1,89 @@
+package io.deephaven.parquet.table.util;
+
+import io.deephaven.base.verify.Assert;
+import org.reactivestreams.Subscriber;
+import org.reactivestreams.Subscription;
+import software.amazon.awssdk.core.async.AsyncResponseTransformer;
+import software.amazon.awssdk.core.async.SdkPublisher;
+
+import java.nio.ByteBuffer;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * An {@link AsyncResponseTransformer} that transforms a response into a {@link ByteBuffer}.
+ * This class is inspired from {@link software.amazon.awssdk.core.internal.async.ByteArrayAsyncResponseTransformer}
```

Comment:
I don't think this comment is necessary. I don't see much similarity in impls IMO, and this one is "pretty straightforward" as far as these APIs are concerned. I'm honestly surprised an impl like this isn't built in...